### PR TITLE
#27 FRONT - Generate severance certificate

### PR DESCRIPTION
**Pull request by yayobyte**

### CODE REVIEW

**Comment by yayobyte** [comment]
@hachesilva Could you please code review this?

File: src/js/app/modules/users/directives/modalSeveranceCertificate.directive.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            bindToController: true
+        };
+    }
+
+    function modalController (GenerateSeveranceFactory){
+        var vm = this;
+        vm.download = {};
+
+        vm.requestDownload = function () {
+            console.log(vm);
```
Comment (hachesilva) [inline src/js/app/modules/users/directives/modalSeveranceCertificate.directive.js]: console.log seems unnecesary here

File: src/js/app/modules/users/controllers/listUsers.controller.js [hunk 2]
```diff
@@ -59,29 +61,40 @@
...
         };
 
         vm.getError = function (error){
-            console.log(error);
-            vm.postStatus = error;
-            vm.postStatus.message = "Error " + error.status + " " + error.statusText + " | Message: " + error.data.summary;
-            vm.postStatus.error = true;
+            vm.download = error;
+            vm.download.message = "Error: " + error.data;
+            console.log(vm.download.message);
```
Comment (hachesilva) [inline src/js/app/modules/users/controllers/listUsers.controller.js:96]: console.log seems unnecesary here

File: src/js/app/modules/users/controllers/listUsers.controller.js [hunk 3]
```diff
@@ -59,29 +61,40 @@
...
                 console.warn('userIdentification not sent');
             }
         };
 
         vm.getSuccess = function (response){
-            vm.postStatus  =  response[0];
-            var fileId = vm.postStatus.tempFileId;
+            var fileId = response.tempFileId;
+            vm.download.message = "Certification created successfully" ;
+            console.log(vm.download.message);
```
Comment (hachesilva) [inline src/js/app/modules/users/controllers/listUsers.controller.js:88]: console.log seems unnecesary here

File: src/js/app/modules/users/directives/modalSeveranceCertificate.directive.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+                userId: vm.userId,
+                subject : vm.subject,
+                amount : vm.amount
+            }, vm.getSuccess, vm.getError);
+        };
+
+        vm.getSuccess = function (response){
+            var fileId = response.tempFileId;
+            vm.download.message = "Certification created successfully" ;
+            console.log(vm.download.message);
```
Comment (hachesilva) [inline src/js/app/modules/users/directives/modalSeveranceCertificate.directive.js]: console.log seems unnecesary here

File: src/js/app/modules/users/directives/modalSeveranceCertificate.directive.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+            vm.download.error = false;
+            window.open(globalConfig.apiEndpoint + globalConfig.apiRoutes.downloadCertification.replace(':file' , fileId));
+        };
+
+        vm.getError = function (error){
+            vm.download = error;
+            vm.download.message = "Error: " + error.data;
+            console.log(vm.download.message);
```
Comment (hachesilva) [inline src/js/app/modules/users/directives/modalSeveranceCertificate.directive.js]: console.log seems unnecesary here

**Comment by hachesilva** [comment]
Several console.log messages need tobe removed, otherwise is good to go.

**Comment by yayobyte** [comment]
Console.logs removed but I left the one when some error happens.

**Pull request merged**